Source/Analytics/Web/src/components/Lightweight/LightweightAreaOverview.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import HealthRiskSelector from '../healthRisk/HealthRiskSelector';
+import CBSNavigation from '../Navigation/CBSNavigation';
+import './LightweightAreaOverview.scss';
+
+export default class LightweightAreaOverview extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div>
+                <CBSNavigation activeMenuItem="analytics/" />
+
+                <h2 className="lightweight header">Light Area Overview</h2>
```

Comment:
With regards to the css class names, I think you should rather have the class "lightweight" on the wrapping div, and then have .lightweight-header, lightweight-description etc.